sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/filters/TargetingFilter.java

```diff
@@ -127,19 +128,43 @@ public boolean evaluate(FeatureFilterEvaluationContext context) {
         updateValueFromMapToList(parameters, USERS);
         updateValueFromMapToList(parameters, GROUPS);
 
-        Map<String, Map<String, Object>> exclusionMap = (Map<String, Map<String, Object>>) parameters
-            .get(EXCLUSION);
-        Map<String, Object> exclusionAsLists = new HashMap<>();
-        if (exclusionMap != null) {
-            exclusionAsLists.put(USERS, exclusionMap.getOrDefault(USERS, new HashMap<String, Object>()).values());
-            exclusionAsLists.put(GROUPS, exclusionMap.getOrDefault(GROUPS, new HashMap<String, Object>()).values());
+        Audience audience;
```

Comment:
Do you think that we can use ObjectMapper to do case-insensitive deserialization here?

Reply:
The issue here is that when Spring ingested the configuration it takes all lists and converts them to `Map` with key = location in the list.